python/ray/monitor.py

```diff
@@ -219,6 +216,16 @@ def _run(self):
             # round of messages.
             time.sleep(AUTOSCALER_UPDATE_INTERVAL_S)
 
+    def update_event_summary(self):
+        # Periodically emit event summaries.
+        avail_resources = self.load_metrics.resources_avail_summary()
+        if avail_resources != self.last_avail_resources:
+            self.event_summarizer.add(
+                "Resized to {}.",
```

Comment:
```suggestion
                "Resized cluster to {}.",
```
OR (might be more accurate since the cluster might have not "resized" in between two calls here).
```suggestion
                "Current cluster size: {}.",
```

Reply:
Using a verb is more consistent with the other event logs.